### ISSUE TITLE
Extension no longer finds global standard installation

### ISSUE THREAD

**Issue by benhunt29** [standard/vscode-standard#19]
It seems this extension can no longer find my global installation of standard. I am using nvm and have tried having ```"standard.nodePath": null,``` and ```"standard.nodePath": "/Users/bhunt/.nvm/versions/node/v4.3.2/bin/standard"``` but both result in the following output:

```
[Info  - 12:00:45 PM] JavaScript Standard Style server is running.
[Info  - 12:00:45 PM] 
Failed to load the JavaScript Standard Style library for the document '<pathToJSFile>'.

To use JavaScript Standard Style in this workspace please install standard using 'npm install standard' or globally using 'npm install -g standard'.
You need to reopen the workspace after installing standard.

Alternatively you can disable JavaScript Standard Style for this workspace by executing the 'Disable JavaScript Standard Style for this workspace' command.
```

I also have the following settings in settings.json

```
"javascript.validate.enable": false,    
    "standard.enable": true,
    "standard.autoFixOnSave": true
```

Any idea what's going on?

**Comment by chenxsan** [comment]
I'm running nvm too :)

<img width="932" alt="screen shot 2017-03-08 at 7 59 05 am" src="https://cloud.githubusercontent.com/assets/1091472/23683666/357d9654-03d5-11e7-9e87-1bac547bc40c.png">

Which version of standard do you have?

**Comment by benhunt29** [comment]
I'm using Standard version 9.0.1, VS Code 1.10.1, and version 1.1 of your extension. It is a little weird, I can get it to work fine on my personal machine, but my work machine just recently quit working. 

If I locally install standard, the extension works fine. It just doesn't seem to want to pick up the global package. Simply running "standard" from the command line with no local package installed works however.

**Comment by benhunt29** [comment]
Issue seems to be fixed after upgrading to VS Code 1.10.2
